priv/catalog/DA114A.yaml

```diff
@@ -0,0 +1,47 @@
+id: DA114A
+name: Corosync rings
+group: Corosync
+description: |
+  Corosync has at least 2 rings configured
+remediation: |
+  ## Abstract
+  It is strongly recommended to add a second ring to the corosync communication.
+
+  ## References
+  Azure:
+
+    - https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/high-availability-guide-suse-pacemaker
+
+  AWS:
+
+    - https://docs.aws.amazon.com/sap/latest/sap-hana/sap-hana-on-aws-cluster-configuration.html
+
+  GCP:
+
+    - https://cloud.google.com/solutions/sap/docs/sap-hana-ha-config-sles
+
+  SUSE / KVM:
+
+   - https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/#id-example-for-etccorosynccorosync-conf
+   - section 9.1.3 in https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/#id-adapting-the-corosync-and-sbd-configuration
```

Comment:
This `section 9.1.3` doesn't make many sense. We should update the link to point to it exactly.
Maybe this: https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/#id-checking-the-corosync-configuration

At least the current link doesn't take you to any particular chapter. Maybe the url has been changed...

Reply:
No clue :sweat_smile: I ported the remediation text from the [current check](https://github.com/trento-project/runner/blob/main/runner/ansible/roles/checks/1.1.9/defaults/main.yml) in the runner.

Maybe @pirat013 knows better what should go there.

Reply:
yep, seems the Permalink had changed, so https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/#id-checking-and-adapting-the-corosync-and-sbd-configuration should be the 'desired' one.

Reply:
Thanks, changed.